### PR TITLE
Fix font size and colour issue

### PR DESCRIPTION
**Pull request by jon-stevens**
reddy

### CODE REVIEW

File: toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss [hunk 1]
```diff
@@ -25,15 +25,14 @@
...
-		font-size: .75rem;
 	}
 
 	&__link {
 		padding: 7px 12px;
 
 		&:hover {
 			background-color: map-get($context--colors, universal-blue);
-			color: map-get($context--colors, white);
+			color: map-get($context--colors, 'white');
```
Comment (aliuk2012) [inline toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss:35]:
```suggestion
			color: color('white');
```

File: toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss [hunk 2]
```diff
@@ -25,15 +25,14 @@
 
 	&__content-item {
 		padding: 0 12px;
```
Comment (aliuk2012) [inline toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss:27]: 🤔  Sorry I know you didn't add this in as part of these changes but I'm wondering if this padding should actually also be placed in core.scss like the font size is.

Comment (jon-stevens) [reply toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss:27]: No that's not required for the core version.

**Review by aliuk2012 — APPROVED** [review]

**Comment by jon-stevens** [comment]
Thanks Al 👍

**Pull request merged**